src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -8,11 +8,13 @@ import ClientMetricReport from './ClientMetricReport';
 import Direction from './ClientMetricReportDirection';
 import MediaType from './ClientMetricReportMediaType';
 import GlobalMetricReport from './GlobalMetricReport';
+import RawMetricReport from './RawMetricReport';
 import StreamMetricReport from './StreamMetricReport';
 
 export default class DefaultClientMetricReport implements ClientMetricReport {
   globalMetricReport: GlobalMetricReport = new GlobalMetricReport();
   streamMetricReports: { [id: number]: StreamMetricReport } = {};
+  rawMetricReport: RawMetricReport = new RawMetricReport();
```

Comment:
How is this one get set?

Reply:
https://github.com/aws/amazon-chime-sdk-js/blob/e67ada29b03207aec0e9b68f245963dcfb0bab7b/src/statscollector/DefaultStatsCollector.ts#L459